muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace-materials-management.js

```diff
@@ -562,6 +562,9 @@
       var fileId = data._response.result.fileId;
       var fileName = data.files[0].name;
       var contentType = data.files[0].type;
+      if (!contentType || contentType == '') {
```

Comment:
both sides of or yield to same result, so only if (!contentType) would be suffient, but using var contentType = data.files[0].type||'application/octet-stream'; would be even better.